ui/src/app/pages/workspace/workspace-edit.spec.tsx

```diff
@@ -294,16 +303,20 @@ describe('WorkspaceEdit', () => {
   });
 
   it('does not display the access tier dropdown when multiple tiers are not available', async() => {
-    cdrVersionStore.set( {tiers: [cdrVersionTiersResponse.tiers[0]]});
```

Comment:
this change reflects the new behavior.  IMO this still checks the "available tiers" by a different definition